foundry_example/garden_publish.ipynb

```diff
@@ -110,12 +96,12 @@
    "outputs": [],
    "source": [
     "# create a garden and add the pipeline to it, then publish it to the search index\n",
-    "garden = client.create_garden(pipeline.authors,\n",
-    "                              pipeline.title,\n",
-    "                              pipelines=[pipeline],\n",
-    "                              pipeline_ids=[pipeline.doi],\n",
-    "                              description=pipeline.description,\n",
-    "                              tags=pipeline.tags)\n",
+    "garden = client.create_garden(registered.authors,\n",
+    "                              registered.title,\n",
+    "                              pipelines=[registered],\n",
```

Comment:
Let's cut this line -- we want people to add pipelines by just the DOI anyway, and  pydantic (I assume, if it wasn't an error) ignores the `pipelines=[registered]` parameter because that's just a regular `@property` instead of a pydantic `Field` attribute.